### PR TITLE
Make the module usable via npm / browserify

### PR DESCRIPTION
**Pull request by lemmingworks**
Hi,

I've just added an index.js file to the package which can make the page available via npm and therefore browserify. Example shown in update to readme.

I just have my cloned version in my private npm repository at the moment, so to make the change work, the module would have to be made publicly available on the npm repository, but I didn't want to do that without your permission.

The package.json could do with updating with your details and version numbers as well. I'm happy to make the updates if you let me know what the details should be.

Thanks
Joe

### CODE REVIEW

**Comment by aaronbeall** [comment]
@ninjatronic Please accept this pull request, I need to use angular-base64 in a Webpack project, and found that `import base64 form "angular-base64"` does not import anything meaningful (most angular libraries export the angular module name, ie `"base64"` in this case, or the entire `angular.module` object.)

File: package.json [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
+    "grunt-contrib-copy": "",
+    "grunt-contrib-jshint": "",
+    "grunt-contrib-watch": "",
+    "grunt-contrib-concat": "",
+    "grunt-contrib-uglify": "",
+    "grunt-notify": "",
+    "grunt-strip": "",
+    "grunt-text-replace": ""
+  },
+  "name": "@lemming/angular-base64",
```
Comment (ninjatronic) [inline package.json]: Please alter this to `@ninjatronic/angular-base64`

**Comment by ninjatronic** [comment]
Hi @joelemmer,

Please check the comments on the `package.json` changes and change as appropriate. I'll accept the PR once done.

Thanks

**Comment by jwanglof** [comment]
@joelemmer How's the progress on making the changes @ninjatronic wants? Would really need this for my new project =)

**Comment by joelemmer** [comment]
@ninjatronic @jwanglof I've made those amendments to the package.json. Sorry for the delay. You should be able to accept this now. Regards. Joe

**Comment by jwanglof** [comment]
@ninjatronic How about accepting this PR?

**Comment by ninjatronic** [comment]
Been away a long time. Thanks for the PR

**Pull request merged**